development/tools/cmd/configuploader/README.md

```diff
@@ -0,0 +1,22 @@
+# Config Uploader
+
+## Overview
+
+This command is responsible for uploading Prow configuration.
+
+## Usage
```

Comment:
If this tool has to be used for production cluster, please update documentation how to configure production cluster

Reply:
Updated